LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2020 Urban Computing Foundation
+Copyright Vis.gl contributors.
```

Comment:
nit: any opinion on if we want it lowercase or capitalized? It's whatever we prefer